tests/dash/dash_db.py

```diff
@@ -249,3 +276,15 @@ def create_routing_type(self, routing_type, attr_maps: dict):
 
     def remove_routing_type(self, routing_type):
         del self.app_dash_routing_type_table[str(routing_type)]
+
+    def create_meter_policy(self, meter_policy, attr_maps: dict):
```

Comment:
Please use the generic `set_app_db_entry` and `remove_app_db_entry` methods instead